features/loopMessages/index.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * ------------------------------
+ * Loop Messages
+ * ------------------------------
+ * The bot will keep a message as the last one in a channel, with an interval between each post,
+ * to keep important information more visible to users
+ *
+ * How it works:
+ * At startup and repeatedly by a given interval, for each loop message, the bot will:
+ * 1. Get the target channel of the loop message
+ * 2. Fetch the last 50 messages of the channel
+ * 3. If the last message is already the message we want to loop, do nothing (we don't want to spam the channel)
+ * 4. Delete all loop messages (so the channel isn't filled with bot messages)
+ * 5. Send the loop message
+ */
+
+const loopMessages = require("./messages.js");
+
+// time between each check to send a new loop message
+const INTERVAL_TIME = 5 * 60 * 1000; // 5 minutes (ms)
+
+function getContent(loopMessage) {
+  return loopMessage.trim();
+}
+
+function isMessageFromBot(bot, message) {
+  return message.author.id === bot.user.id;
+}
+
+function isLoopMessage(messageToCheck, loopMessage, bot) {
+  return (
+    isMessageFromBot(bot, messageToCheck) &&
+    messageToCheck.content.trim() === getContent(loopMessage.content)
+  );
+}
+
+async function sendLoopMessage(client, loopMessage) {
+  const channel = client.channels.get(loopMessage.channelId);
+  const channelMessages = await channel.fetchMessages({ limit: 50 });
```

Comment:
I'm guessing if the loop message is past 50 messages, it probably doesn't matter if we delete it or not 😛 I'm not exactly sure how the reactiflux website is parsing the job posts, not sure if it would pick up the bots messages for the jobs channel specifically.

Reply:
> I'm guessing if the loop message is past 50 messages, it probably doesn't matter if we delete it or not

Yeah exactly.

> I'm not exactly sure how the reactiflux website is parsing the job posts, not sure if it would pick up the bots messages for the jobs channel specifically.

I was gonna say that the bot only parses the messages that contain the tags `([FOR HIRE]`, `[HIRING]`, etc) but I just realized that they are in the message content lol
Great catch, I will change the jobs script to ignore the messages from the bot